### PR TITLE
issue-2936 - Adding failing test case

### PR DESCRIPTION
**Pull request by mitermayer**
WIP

This has only the broken tests

### CODE REVIEW

**Comment by azz** [comment]
The failing test seems to be working correctly, it's finding the `.prettierrc` file in `tests_integration/cli/config/` and preferring that to the CLI args you are passing.

**Comment by azz** [comment]
Wait, you're right, they're supposed to be merged...

File: tests_integration/__tests__/with-config-precedence.js [hunk 1]
```diff
@@ -58,6 +58,52 @@ test("CLI overrides gets ignored when config exists with --config-precedence pre
...
+    ], {
+      input: ".t { background: red; }"
+  });
+
+  expect(output.stdout).toMatchSnapshot();
+  expect(output.status).toEqual(0);
+});
+
+test("with --config-precedence prefer-file and --stdin", () => {
+  const output = runPrettier("cli/config", [
```
Comment (azz) [inline tests_integration/__tests__/with-config-precedence.js]: This passes if you change `"cli/config"` to `"cli"`.

**Comment by mitermayer** [comment]
You are totally right!

**Pull request closed without merging**